src/main/scala/com/campudus/tableaux/controller/DemoController.scala

```diff
@@ -0,0 +1,62 @@
+package com.campudus.tableaux.controller
+
+/**
+ * Controller for resetting the database.
+ * Mostly this is need for Demo & Test data.
+ */
+import com.campudus.tableaux.ArgumentChecker._
+import com.campudus.tableaux.database.Tableaux._
+import com.campudus.tableaux.database._
+import com.campudus.tableaux.database.structure.CreateColumn
+import com.campudus.tableaux.helper.HelperFunctions._
+import com.campudus.tableaux.helper.StandardVerticle
+import org.vertx.scala.core.json.Json
+import org.vertx.scala.platform.Verticle
+
+import scala.concurrent.Future
+
+class DemoController(val verticle: Verticle, val database: DatabaseConnection) extends DatabaseAccess with StandardVerticle {
+  val demoJson = Json.obj(
```

Comment:
I'll open an issue for putting the demo data out of the code and have it in a separate file. I don't really like that we're writing all this in Scala... ;)